src/app/data/components/data-browser/data-browser.component.scss

```diff
@@ -87,5 +87,5 @@ input[type="number"] {
 }
 
 .tooltip-label::before {
-  z-index: 10;
+  z-index: 50;
```

Comment:
if we want to use our own class for this why not also include the display relative part which is a prerequisite for this to work?

Reply:
I think there is no need for own class tooltip-label, we can use the tooltip and add the z-index. there. I will replace it as part of this PR.